src/main/java/com/ericsson/ei/subscriptionhandler/SendMail.java

```diff
@@ -107,6 +111,10 @@ public void sendMail(String receiver, String mapNotificationMessage) throws Mess
         return emailAdd;
     }
 
+    public String getSubject(String subject, String emailSubject) {
```

Comment:
String subject already exists in the scope of the class as an instance variable, does it really need to be explicitly sent to this method?